charts/greptimedb-cluster/templates/secret.yaml

```diff
@@ -2,7 +2,11 @@
 {{- if .Values.objectStorage.credentials }}
 apiVersion: v1
 metadata:
-  name: {{ .Release.Name }}-secret
+  {{- if .Values.objectStorage.credentials.secretName }}
```

Comment:
suggestion: can we use the following code to make it short:

```
name: {{ default "storage-credentials" .Values.objectStorage.credentials.secretName }}
```

Reply:
done